src/pages/Gallery/components/Category/category.styles.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components'
+
+export const CategoriesList = styled.ul`
+  .categories {
+    width: 50%;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: nowrap;
+    align-content: center;
+    justify-content: center;
+    align-items: center;
+    justify-content: space-around;
+    border: 1px solid white;
+    margin: 0 auto 100px;
```

Comment:
```suggestion
  .categories {
    width: 50%;
    flex-wrap: nowrap;
    ${FlexAlignCenter}
    justify-content: space-around;
    border: 1px solid white;
    margin: 0 auto 100px;
```
이렇게 해도 똑같을 거 같은데 한번 해볼래요?!